client/components/navbar/navbar.html

```diff
@@ -15,19 +15,49 @@
             <a ng-href="{{item.link}}">{{item.title}}</a>
         </li>
         <li ng-show="isLoggedIn()" ng-class="{active: isActive('/attend')}"><a href="/attend">Attend</a></li>
-        <li ng-show="isMentor()" ng-class="{active: isActive('/mentor')}"><a href="/mentor">Mentor</a></li>
-        <li ng-show="isAdmin()" ng-class="{active: isActive('/admin')}"><a href="/admin">Admin</a></li>
-        <li ng-show="isLoggedIn() && (isMentor() || hasSmallGroup())"><a ng-href='/smallgroup'>Small Group</a></li>
-        <li ng-show="isAdmin()" ng-class="{active: isActive('/admin/classcontrol')}"><a href="/admin/classcontrol">Class Control</a></li>
         <li ng-show="displayURP"><a href='http://urp.rcos.xyz/'>URP Form</a></li>
+        <li ng-show="isMentor()" class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+                Mentor <span class="caret"></span></a>
+            <ul class="dropdown-menu">
+                <li ng-show="isMentor()" ng-class="{active: isActive('/mentor')}">
+                    <a href="/mentor">Verify Attendance</a>
+                </li>
+                <li ng-show="isLoggedIn() && (isMentor() || hasSmallGroup())">
```

Comment:
might not want to have small groups under mentor tab... anyone in a small group should be able to view the page